smaily.php

```diff
@@ -6,11 +6,11 @@
  *
  * @wordpress-plugin
  * Plugin Name:       Smaily
```

Comment:
Change the plugin name to "Smaily for Wordpress". We want our plugin slug to be `smaily-for-wordpress`, and then this file should be named `smaily-for-wordpress.php` (instead of `smaily.php`).